observable-app/pom.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>observability-kit</artifactId>
+        <version>2.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>observe</artifactId>
+    <name>Observe application</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <maven.install.skip>true</maven.install.skip>
+        <exampledata.version>6.2.0</exampledata.version>
+        <vaadin.version>24.0-SNAPSHOT</vaadin.version>
```

Comment:
Probably we don't need this property if we use explicit Flow dependencies with the parent `flow.version` property.